app/src/main/res/values/colors.xml

```diff
@@ -20,5 +20,6 @@
     <color name="color_indogo_acceent_700">#304FFE</color>
     <color name="light_blue_900">#01579B</color>
     <color name="color_black_translucent">#80000000</color>
+    <color name="color_black">#1a1a1a</color>
```

Comment:
WARNING: [UnusedResources] The resource `R.color.color_black` appears to be unused
 Unused resources make applications larger and slow down builds.